.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
I think that in general, it will not be possible to refactor to appease the cop. It's currently 0 for 2 - a 100% false positive rate. In my estimation, you will not get better than 50% out of this cop.

Fixing linter warnings in CI is annoying and it's especially annoying if it's not about actual issues. Unless you're willing to make CI block merges into master, the likely result is people will just ignore the failing build, at which point your CI is completely useless.

So again,
> The way I see it, there merely happen to be no other unused kwargs. But if there were, we'd just disable the cop there too.

Since there is no way to fix most (in this case, all) of the issues raised by this cop, it should be disabled.